docker/docs/builder/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# docker build -t yandex/clickhouse-docs-build .
+FROM ubuntu:20.04
+
+ENV LANG=C.UTF-8
+
+RUN sed -i 's|http://archive|http://ru.archive|g' /etc/apt/sources.list
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install --yes --no-install-recommends \
+            python3-setuptools \
+            virtualenv \
+            wget \
+            bash \
+            python \
+            curl \
+            python3-requests \
+            sudo \
+            git \
+            openssl \
+            python3-pip \
+            software-properties-common \
+            language-pack-zh* \
+            chinese* \
+            fonts-arphic-ukai \
+            fonts-arphic-uming \
+            fonts-ipafont-mincho \
+            fonts-ipafont-gothic \
+            fonts-unfonts-core \
+            xvfb \
+            nodejs \
+            npm \
+            openjdk-11-jdk \
+    && pip --no-cache-dir install scipy \
+    && apt-get autoremove --yes \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN wget 'https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.focal_amd64.deb'
```

Comment:
I prefer to completely get rid of server-side pdf generation.
It does not work properly anyway (Chinese fonts are mangled and no one is being able to fix it in years). It also painfully slow and it's one of the reasons our documentation is so bad.

Instead we can trigger 'print' in single page version in browser to allow "print to pdf".